tests/unit_tests/steps/test_compile_fortran.py

```diff
@@ -8,13 +9,15 @@
 from fab.constants import BUILD_TREES, OBJECT_FILES
 
 from fab.dep_tree import AnalysedFile
-from fab.steps.compile_fortran import CompileFortran
+from fab.steps.compile_fortran import CompileFortran, _get_compiler_version
```

Comment:
Is `get_compiler_version` supposed to be private (in which case don't import it) or is it supposed to be public, in which case loose the underline.

Or is it supposed to be private but testing throws up these problems. In which case "Kiff, we have a conundrum."

Reply:
It's supposed to be private to the compile step and it's tests, cap'n.